libs/postgres_ffi/src/xlog_utils.rs

```diff
@@ -53,6 +53,8 @@ pub type XLogRecPtr = u64;
 pub type TimeLineID = u32;
 pub type TimestampTz = i64;
 pub type XLogSegNo = u64;
+pub type XidCSN = u64; 
+pub type SnapshotCSN = u64;
```

Comment:
It seems these types can be generated through bindgen by adding `utils/snapshot.h` in `bindgen_deps.h` and enable the types in the `build.rs`

Reply:
I looked into it earlier, but it didn't seem to work. From the description, I realized that it translates structures, not typedefs. The whitelisting is only for structures and constants; this does not work with typedefs. Other types have also been redefined in xlog_utils.rs, for example XLogRecPtr is defined explicitly, and also xlogdefs.h is not included in bindgen_deps.h

Reply:
I see. That makes sense.